xarray/core/dataarray.py

```diff
@@ -2216,19 +2216,44 @@ def interp(
         coords: Mapping[Any, Any] | None = None,
         method: InterpOptions = "linear",
         assume_sorted: bool = False,
+        reduce: bool = True,
         kwargs: Mapping[str, Any] | None = None,
         **coords_kwargs: Any,
     ) -> Self:
-        """Interpolate a DataArray onto new coordinates
+        """
+        Interpolate a DataArray onto new coordinates.
+
+        Performs univariate or multivariate interpolation of a Dataset onto new coordinates,
+        utilizing either NumPy or SciPy interpolation routines.
+
+        When interpolating along multiple dimensions, the process attempts to decompose the
+        interpolation into independent interpolations along one dimension at a time, unless
+        `reduce=False` is passed. The specific interpolation method and dimensionality
+        determine which interpolant is used:
+
+        1. **Interpolation along one dimension of 1D data (`method='linear'`)**
```

Comment:
Let's move these to the `Notes` section: https://numpydoc.readthedocs.io/en/latest/format.html#notes

Reply:
Do you mean the list of interpolants or the description of the reduce behavior (now automatic/no keyword)? Thanks!

Reply:
The list of interpolants